todo-backend/README.adoc

```diff
@@ -351,16 +439,34 @@ $ oc new-app postgresql-ephemeral \
 This will create a PostgreSQL database named `todos` on OpenShift that can be accessed on the port `5432` on the service `todos-db`.
 We don't need to copy the credentials to connect to the database as we will retrieve them later using the `todos-db` secret that was created
 when the database is deployed.
+endif::[]
 
-ifndef::ProductRelease,EAPXPRelease[]
+ifndef::ProductRelease[]
 === Build and Deploy the Backend on OpenShift with Bootable Jar
 
+==== Deploy a PostgreSQL Database on OpenShift
+
+[source,options="nowrap"]
+----
+$ oc new-app postgresql-ephemeral \
+   -p DATABASE_SERVICE_NAME=todos-db \
+   -p POSTGRESQL_DATABASE=todos
+----
+
+This will create a PostgreSQL database named `todos` on OpenShift that can be accessed on the port `5432` on the service `todos-db`.
+We don't need to copy the credentials to connect to the database as we will retrieve them later using the `todos-db` secret that was created
+when the database is deployed.
+
+==== Build and Deploy the Backend on OpenShift with Bootable Jar using the Helm Chart
+
 The backend will be built and deployed on OpenShift with a Helm Chart for {productName}.
 
+ifndef::ProductRelease,EAPXPRelease[]
+
 [source,options="nowrap",subs="+attributes"]
 ----
-$ helm install todo-backend --set build.ref={WildFlyQuickStartRepoTag} -f https://raw.githubusercontent.com/wildfly/wildfly-charts/main/examples/todo-backend/todo-backend-bootable-jar.yaml wildfly/wildfly
-NAME: todo-backend
+$ helm install todo --set build.ref={WildFlyQuickStartRepoTag} -f https://raw.githubusercontent.com/wildfly/wildfly-charts/main/examples/todo-backend/todo-backend-bootable-jar.yaml wildfly/wildfly
```

Comment:
Here the situation is an opposite. IMO the `build.ref` should point to the state of `todo-backend` QS before this PR updated one. What is the best value for that?

Reply:
Not sure I understand :(